docs/advanced-configuration.rst

```diff
@@ -0,0 +1,279 @@
+Advanced Configuration
```

Comment:
this whole file seems to mostly be things we might want to support one day, but don't... should it be in `unfinished/` ?

Reply:
yes.